lib/TBDGen/TBDGen.cpp

```diff
@@ -338,9 +343,21 @@ void TBDGenVisitor::visitExtensionDecl(ExtensionDecl *ED) {
 }
 
 void TBDGenVisitor::visitProtocolDecl(ProtocolDecl *PD) {
-  if (!PD->isObjC())
+  if (!PD->isObjC()) {
     addSymbol(LinkEntity::forProtocolDescriptor(PD));
 
+    if (!PD->hasFixedLayout()) {
```

Comment:
Can we not do the same "minimal resilience" query here that we do in IRGen?

Reply:
`isResilient()` is defined in IRGenModule, which we don't have an instance of here, but I can use the form of `hasFixedLayout()` that takes a module and resilience expansion instead. Perhaps `hasFixedLayout()` should be renamed to `isResilient()`, for consistency between AST and IRGen?

Reply:
Yeah, it'd be nice to have consistent terminology across the compiler.